bench/README.md

```diff
@@ -16,112 +16,114 @@ In another configuration, we benchmark the worst-case scenario performance of
 `JsonReader.Either` by deserializing an array with elements of type
 `MultiPolygon`. We measure the following:
 
-    BenchmarkDotNet=v0.13.2, OS=Windows 11 (10.0.22621.819)
+    BenchmarkDotNet=v0.13.2, OS=Windows 11 (10.0.22621.963)
```

Comment:
Note to update the benchmarks after the review, as the benchmarks take a while to execute.